### DISCUSSION TITLE
How to disable transcoding?

### DISCUSSION THREAD

**Discussion by ghost** [jellyfin/jellyfin#5777] [Q&A]
When I stream 1080p videos from Jellyfin to my fireTv stick it lugs each minute or so, work well with 720p and lower.
I guess it's due to transcoding, I may be wrong.

How can I solve the issue?

**Comment by hurricane-dorian** [comment]
jellyfin only transcodes if the device is unable to play a particular format.  Often the culprit is audio or subtitles. EG: If the stream is DTS but your audio-system can only decode Dolby then jellyfin will transcode.
Check the logs, jellyfin gives a reason why it is transcoding the stream.  Disabling transcoding would mean that your device would no longer be able to play the stream.  
The simplest solution is of course to just pause for a few minutes to give ffmpeg time to build a buffer.
You could also try to enable hardware transcoding. Intel quicksync works great even on my lowly J4105

**Comment by ghost** [comment]
Pausing works, thanks

**Comment by dmitche3** [comment]
LOL.  Sorry but that's a pretty sad solution.  You might as well pause the playback until enough of the stream is buffered.

**Comment by eingemaischt** [comment]
BTW: I activated playback through libVLC in the jellyfin app on the Fire TV Stick. It can plays way more files without transcoding than  exoplayer.
